config/jest/automock.js

```diff
@@ -4,5 +4,8 @@ jest.mock('electron');
 jest.mock('fs');
 jest.mock('../../src/utils/Environment');
 jest.mock('../../src/utils/uuid');
+jest.mock('../../src/utils/CSSVariables');
+jest.mock('../../src/utils/uuid');
+jest.mock('../../src/ui/utils/CSSVariables');
```

Comment:
Jest treats the two CSSVariable mocks duplicates anyway (see warning when test runs); I don't think automocking here does anything.

We could clean up the mock includes, but this is probably an indication that we should remove the duplication in code.